Updated examples

kylemas opened the pull request.

McTano commented on src/examples/completeExample.json:
```diff
@@ -0,0 +1,24 @@
+    {
+      "nodeType": "contradiction",
+      "formulas": [],
+      "rule": ["X"]
```
Is this supposed to be a correct example?
If so, there's a contradiction node closing the tree but no contradictory formulas, so this is incorrect.
If not, maybe change the name to make it clear this is supposed to be an incorrect example.

McTano commented on src/examples/incompleteExample.json:
```diff
@@ -0,0 +1,29 @@
+    {
+      "nodeType": "finished",
+      "formulas": [],
+      "rule": ["O"]
```
rule should be a string, not an array.

McTano commented on src/examples/invalidExample.json:
```diff
@@ -0,0 +1,44 @@
+    {
+      "nodeType": "contradiction",
+      "formulas": [],
+      "rule": ["X"]
```
See comments on previous example.

McTano commented on src/examples/completeExample.json:
```diff
@@ -0,0 +1,24 @@
+    {
+      "value": " P&D",
+      "row": 2,
+      "resolved": false
```
For a "complete" example, all non-atomic nodes should be resolved.

McTano replied:
@kylemas
This formula still needs to be resolved

kylemas commented:
> It seems like you're having some trouble with this. Maybe I haven't explained clearly what we need from these examples.
> It looks like you're still creating these by hand. I think the best way to do this is to work through an example fully on the widget, and just copy the json that gets displayed below the tree. I'll do one and send you an example.
> Do some correct examples first and then we'll worry about what kind of incorrect examples we want.
> We can find a time to go over this exercise so we can figure out where the confusion is coming from.

@McTano 
Hey Tristan! Sorry about that, looks like I misunderstood what was needed. I worked through the examples and used the JSON output at the bottom of the page and added it to the files. I fixed up the correct, invalid, and complete examples but not the incorrect example.

McTano commented:
@kylemas 
This looks good now except for the unresolved formula on `completeExample`.

Please fix that and give the files more specific names. Then you can merge.
